triframe/phases/actor.py

```diff
@@ -28,10 +28,14 @@
 )
 from utils.logging import log_warning
 from utils.phase_utils import (
+    add_dummy_user_message,
     add_usage_request,
+    get_thinking_blocks,
     run_phase,
 )
 
+CLAUDE_THINKING_MODELS = ("claude-3-7-sonnet-20250219",)
```

Comment:
Since this is only defined in triframe, I think that means that either modular:
* is broken
* doesn't support thinking